CreeDictionary/API/models.py

```diff
@@ -186,9 +187,9 @@ def get_absolute_url(self) -> str:
             "cree-dictionary-index-with-lemma", kwargs={"lemma_text": self.text}
         )
         if self.homograph_disambiguator is not None:
-            lemma_url += f"?{self.homograph_disambiguator}={getattr(self, self.homograph_disambiguator)}"
+            lemma_url += f"?{self.homograph_disambiguator}={quote(getattr(self, self.homograph_disambiguator))}"
 
-        return iri_to_uri(lemma_url)
+        return lemma_url
```

Comment:
query params were not escaped correctly.

Specifically, the plus sign in `?analysis=xx+xx+xx+xx` wasn't escaped, which affected the ayaw case